kernel32_data_windows.go

```diff
@@ -13,3 +13,7 @@ func globalFreeWrapper(ptr *uint16) {
 		_, _, _ = globalFree.Call(uintptr(unsafe.Pointer(ptr)))
 	}
 }
+
+func rTrue(r uintptr) bool {
```

Comment:
This function seems pointless.  Why not just if r == 1 { ... } inline rather than call this

Reply:
This is already in two places and the windows return values are quite cryptic :).